docs/src/app.tsx

```diff
@@ -19,6 +19,7 @@ class App extends React.Component<any, ThemeContextData> {
     return (
       <ThemeContext.Provider value={this.state}>
         <Provider
+          as={React.Fragment}
```

Comment:
This is temporary hotfix for Docsite when switching themes for examples. We should fix theme switching to apply to examples only, not to the whole docsite.